src/Http/Request/RequestExecutor.ts

```diff
@@ -69,6 +72,7 @@ export class RequestExecutor extends Observable implements IRequestExecutor {
       "Raven-Client-Version": "4.0.0-beta",
```

Comment:
This should match the value from `package.json`. Can we pull it in runtime from there or embed it in a constant and then use here?

Reply:
as version is changed from release to release i pull it on each build